optimum/intel/ipex/modeling_base.py

```diff
@@ -353,12 +353,12 @@ def _reorder_cache(self, *args, **kwargs):
     def prepare_inputs_for_generation(self, *args, **kwargs):
         return self.model.prepare_inputs_for_generation(*args, **kwargs)
 
-    def _supports_num_logits_to_keep(self) -> bool:
+    def _supports_logits_to_keep(self) -> bool:
```

Comment:
also we can keep `_supports_num_logits_to_keep` until versions lower than 4.49 are deprecated (just calling `_supports_logits_to_keep` until then + add a comment to remove once only > v4.49)

Reply:
transformers 4.49 has already deprecated `_supports_num_logits_to_keep`, the only way is keep both...